Calypso-SystemTools-FullBrowser.package/ClyFullBrowser.class/instance/projectSelection.st

```diff
@@ -1,3 +1,3 @@
 accessing
 projectSelection
-	^packageView selection asSelectedParentsOf: ClyManagedProject
\ No newline at end of file
+	^packageView selection asSelectedParentsOf: self projectClass
```

Comment:
If we don't have known superclass for all possible projects we can't implement this method this way

Reply:
Well, before all projects had to be INSTANCES of a single class.
Right now, projects have to be SUB-INSTANCES. This allows us to use inheritance. This is not perfect (I don't like checking by class at all) but is already an improvement on the previous state.